schema/bom-1.6.schema.json

```diff
@@ -4441,6 +4441,11 @@
           "title": "Confidence",
           "description": "The overall confidence of the evidence from 0 - 1, where 1 is 100% confidence."
         },
+        "concludedValue": {
```

Comment:
Isn't the concluded value already set for the given attribute? We are now creating an opportunity for an edge case where the concluded value could be different from the actual value for the given attribute.

Reply:
> Isn't the concluded value already set for the given attribute?

No, this is an omission. 

> We are now creating an opportunity for an edge case where the concluded value could be different from the actual value for the given attribute.

Yes and no.  Yes, it it possible to have a purl for example in the `evidence.identity.method.value` field and it can be different than the `evidence.identity.concludedValue`. This is by design, as each method has a confidence. So you could have a high and low confidence purl, and your concludedValue might be the high confidence value.

If the `evidence.identity.method.value` is a hash for example, then the `concludedValue` might be a purl based on a dictionary lookup in a table for that hash value.  So in this case, the value and the concludedValue would be talking about two different things.

In either case, the `concludedValue` substantiates the identity in `component`. These two values should be the same, yes. But the `concludedValue` may be low confidence, therefore it instructs the end user that the asserted identity in `component` may be incorrect.

One common use case for doing it this way is for decoupling the SBOM and the evidence.  With the use of an external reference of type `evidence` in the SBOM, the SBOM could point to a BOM containing just the evidence for a given component.